IFC4x3/Properties/t/TensileStrength_1f$78NWsPFR92msngecn7H/Documentation.md

```diff
@@ -0,0 +1 @@
+Cylindrical rock specimen, extended along two diametrically opposed generatrices. This test piece is placed between two plates so that the two generators are in contact with the two plates. An increasing force is applied until the test piece breaks.
```

Comment:
This is a description of the process to determine TensileStrength, not the property.

Reply:
Yes. See also next comment.